lib/keeper/token.rb

```diff
@@ -6,13 +6,12 @@ class Token
     # @param private_claims [Hash] the custom claims to encode
     def initialize(private_claims = {})
       @claims = {
-        iss: Keeper.configuration.issuer,               # issuer
-        aud: Keeper.configuration.audience,             # audience
-        exp: Keeper.configuration.expiry.from_now.to_i, # expiration time
-        nbf: DateTime.now.to_i,                         # not before
-        iat: DateTime.now.to_i,                         # issued at
-        jti: SecureRandom.uuid                          # JWT ID
-      }.merge(private_claims)
+        nbf: DateTime.now.to_i, # not before
+        iat: DateTime.now.to_i, # issued at
+        jti: SecureRandom.uuid  # JWT ID
+      }
+      @claims.merge!(Keeper.configuration.base_claims)
+      @claims.merge!(private_claims)
```

Comment:
they is probably a cooler ruby meta way to do this rather than 2 `merge!` calls. I'd like to see some hardcore meta shit happen here.